execution/src/execution.rs

```diff
@@ -188,8 +189,7 @@ impl<R: ExecutionRpc> ExecutionClient<R> {
         payload: &ExecutionPayload,
         index: usize,
     ) -> Result<Option<Transaction>> {
-        let tx = payload.transactions[index].clone();
-        let tx_hash = H256::from_slice(&keccak256(tx));
+        let tx_hash = keccak256(payload.transactions[index].deref()).into();
```

Comment:
Panics of `index out of bounds` could be caused in this line.

I **really** don't like the code style for helios, too many unchecked panics.